### PR TITLE
Added tvOS simulator support

### PR DESCRIPTION
**Pull request by simongranger**

### CODE REVIEW

**Comment by simongranger** [comment]
plus some refactoring

**Comment by LorneFlindall** [comment]
Is it necessary to name it BlueSky at all. Or can it be made more general and just pass in the name of the platform you want. That way the code could be reused with little effort

**Comment by LorneFlindall** [comment]
Update sampleCaps to reflect new changes i.e FullReset and udid.  But for ease of understanding add in comments for app location, and possible options for all

File: lib/ios-deploy.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import { exec } from 'teen_process';
+import { fs } from 'appium-support';
+import logger from './logger';
+import { retryInterval } from 'asyncbox';
+
+const IOSDEPLOY_PATH = `ios-deploy`;
+
+class IOSDeploy {
```
Comment (pmyoui) [inline lib/ios-deploy.js]: I'm wondering if this should extend from the basedriver

Comment (pmyoui) [reply lib/ios-deploy.js]: Unless Im not understanding what this is for

File: README.md [hunk 2]
```diff
@@ -84,10 +84,21 @@ Support added in 5.0+
 | automationName       | `YouiEngine`            |
 | deviceName           | `<cannot be blank>`     |
 | platformName         | `YItvOS`                |
-| udid<sup>1</sup>     | `<device’s udid>`       |
+| udid                 | `<device’s udid>`       |
 | youiEngineAppAddress | `<device’s IP address>` |
 
-<sup>1</sup> Optional. Required if multiple devices are connected to server.
+
+#### BlueSky
```
[the remark this answers is not in the document]
Comment (simongranger) [reply README.md]: both

File: lib/ios-deploy.js (hunk 1 above)
Comment (simongranger) [reply lib/ios-deploy.js]: This is meant to interface with ios-deploy (https://github.com/ios-control/ios-deploy)
It's pretty much copied from appium-xcuitest-driver (https://github.com/appium/appium-xcuitest-driver/blob/master/lib/ios-deploy.js)

**Pull request merged**